spec/czCustomizableSpec.js

```diff
@@ -152,6 +152,42 @@ describe('cz-customizable', function() {
     expect(commit).toHaveBeenCalledWith('WIP: this is my work-in-progress');
   });
 
+  it('should allow edit message before commit', function(done) {
```

Comment:
We can use something like `sed` to change message, but I have decided to keep it simple.